package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "scripts": {
     "start": "node ./bin/www",
+    "debug": "DEBUG=website:* npm start",
     "test": "jest",
     "e2e-test": "jest --config jest.e2e.config.js",
     "lint": "eslint .",
```

Comment:
TODO:
- rename "fix" to "lint:fix"
- add 2 scripts: format:check and format:write

Reply:
Done.